linera-service/tests/end_to_end_tests.rs

```diff
@@ -41,13 +41,9 @@ struct FungibleApp(ApplicationWrapper<fungible::FungibleTokenAbi>);
 
 impl FungibleApp {
     async fn get_amount(&self, account_owner: &AccountOwner) -> Amount {
-        let query = format!(
-            "accounts {{ entry(key: {}) {{ value }} }}",
-            account_owner.to_value()
-        );
+        let query = format!("balance(owner: {})", account_owner.to_value());
```

Comment:
If we plan to revert this changes, we should probably have in a separate PR (before this one)

Reply:
Gonna add `accounts` support to native fungible in a following PR